src/map/linkshell.cpp

```diff
@@ -484,8 +484,9 @@ namespace linkshell
                     LinkshellList.erase(PItemLinkshell->GetLSID());
                 }
             }
-            catch (std::out_of_range&)
+            catch (std::out_of_range& e)
```

Comment:
can we use more than one letter for the exception, so it becomes obvious to newbies what this is? I just don't think "e" will cut it, and that will also stop them from googling what this is.